test_socket.py

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
This file is making the [CI Lint job fail](https://github.com/farbodahm/stateful-stream-processing/actions/runs/5294018130/jobs/9582864185?pr=29), would you mind please run a `black` linter on it? Simply run:
`pip install black`
`black test_socket.py` and then commit the changes.